codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoderTest.java

```diff
@@ -200,20 +201,6 @@ public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
     }
 
     @Test
-    public void dataWriteAfterGoAwayShouldFail() throws Exception {
```

Comment:
> I believe I said it before

I see what your saying about mocks being cumbersome....and we have talked about this before.  If you think it is really beneficial to create a new unit test file which uses more "real" implementations that may be OK.  I just want to make sure it gets tested...and would recommend using a consistent testing strategy unless there is a good reason not to.